test/format/format_config.c

```diff
@@ -335,6 +395,31 @@ config_table(TABLE *table, void *arg)
     if (TV(BTREE_VALUE_MIN) > TV(BTREE_VALUE_MAX))
         testutil_die(EINVAL, "btree.value_min may not be larger than btree.value_max");
 
+    if (GV(RUNS_PREDICTABLE_REPLAY)) {
+        /*
+         * In predictable replay, force the number of rows in a table to be a manageable size so we
+         * can modify key numbers without problems.
+         */
+        TV(RUNS_ROWS) = WT_MAX(TV(RUNS_ROWS), 2 * LANE_COUNT);
+
+        /*
+         * We don't support some operations in predictable replay.
+         */
+        if (!replay_operation_enabled(TRUNCATE)) {
+            if (config_explicit(table, "ops.truncate") && TV(OPS_TRUNCATE))
+                WARN("turning off truncate for table%" PRIu32 " to work with predictable replay",
+                  table->id);
+            config_single(table, "ops.truncate=0", false);
+        }
+        if (!replay_operation_enabled(MODIFY)) {
+            if (config_explicit(table, "ops.pct.modify") && TV(OPS_PCT_MODIFY))
+                WARN("turning off modify operations for table%" PRIu32
+                     " to work with predictable replay",
+                  table->id);
+            config_single(table, "ops.pct.modify=0", false);
```

Comment:
Alphabetize these two conditionals

Reply:
This one is not changed. Is there a reason we have to check truncate first or could they be alphabetized to check MODIFY and then TRUNCATE?

Reply:
My apologies - I missed your comment(s) here.  I always look at the "Conversations" page in github, and yesterday's comment "Alphabetize these two conditionals" does not appear for me.

Reply:
Fixed.